ovh/resource_cloud_project_gateway_test.go

```diff
@@ -93,6 +83,30 @@ func TestAccCloudProjectGateway(t *testing.T) {
 					resource.TestCheckResourceAttr(resourcePath, "model", "s"),
 				),
 			},
+			{
```

Comment:
missing an `ImportStateVerify: true` here (and possibly some fields in a `ImportStateVerifyIgnore`